### PR TITLE
Loa 350 implement riddle popup styling

### PR DESCRIPTION
**Pull request by Kostanix**

### CODE REVIEW

File: app/src/main/res/layout/dialog_accept_quest_popup.xml [hunk 1]
```diff
@@ -1,96 +1,145 @@
...
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
-    android:background="#62def8">
+    android:background="@drawable/conversation_field_nine">
 
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/main_page_profile_button_container"
+        android:layout_width="100dp"
```
Comment (ayal104) [inline app/src/main/res/layout/dialog_accept_quest_popup.xml:11]: Please fix the IDs.

File: app/src/main/res/layout/dialog_accept_quest_popup.xml [hunk 2]
```diff
@@ -1,96 +1,145 @@
...
-        android:layout_height="120dp"
-        android:layout_alignParentStart="true"
-        android:layout_alignParentTop="true"
-        android:layout_marginStart="20dp"
-        android:layout_marginTop="20dp"
-        android:contentDescription="@string/popup_name_NPC"
-        app:srcCompat="@drawable/circle"
-        tools:layout_marginTop="20dp" />
+        <com.google.android.material.imageview.ShapeableImageView
+            android:id="@+id/mainPage_profileButton"
```
Comment (ayal104) [inline app/src/main/res/layout/dialog_accept_quest_popup.xml:19]: Fix this ID as well.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -57,8 +57,8 @@
     <string name="badgesPage_badge2_background">badge2_background</string>
     <string name="badgesPage_profileButton">badgesPage_profileButton</string>
     <string name="quest_description_text">Beschreibung</string>
-    <string name="npc_name">%1$s will dir eine Quest Anbieten</string>
-    <string name="quest_details">%1$f, %2$f \n Aufgabe: %3$s</string>
+    <string name="npc_name">Von: %1$s</string>
+    <string name="quest_details">\"%1$s\"</string>
     <string name="quest_accept">Accept</string>
```
Comment (ayal104) [inline app/src/main/res/values/strings.xml:62]: Also maybe change Accept and Decline to German here.

**Review by ayal104 — CHANGES_REQUESTED** [review]
See comments :)

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (Kostanix) [reply app/src/main/res/values/strings.xml:62]: Already did in another branch

**Review by ayal104 — APPROVED** [review]
LGTM

**Pull request merged**